x-pack/plugins/enterprise_search/server/routes/app_search/curations.ts

```diff
@@ -13,6 +13,39 @@ export function registerCurationsRoutes({
   router,
   enterpriseSearchRequestHandler,
 }: RouteDependencies) {
+  router.get(
+    {
+      path: '/api/app_search/engines/{engineName}/curations',
+      validate: {
+        params: schema.object({
+          engineName: schema.string(),
+        }),
+        query: schema.object({
+          'page[current]': schema.maybe(schema.number()),
+          'page[size]': schema.maybe(schema.number()),
+        }),
```

Comment:
This is optional for the ent-search API, but we actually always pass this in from Kibana's client-side now. I'm debating removing the `schema.maybe` as a result, any preference from y'all?